### PR TITLE
Eliza code cleanup

### PR DESCRIPTION
**Pull request by leogrim**
@ryanpbrewster @andrewconner

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala [hunk 1]
```diff
@@ -361,16 +378,18 @@ class NotificationDeliveryCommander @Inject() (
...
 
   def setMessageNotificationsRead(userId: Id[User]): Unit = {
     log.info(s"Setting Messaging Notifications as read for user $userId")
-    db.readWrite(attempts = 2) { implicit session => userThreadRepo.markAllRead(userId, Some(false)) }
+    db.readWrite(attempts = 2) { implicit session =>
+      userThreadRepo.markAllRead(userId)
+    }
   }
 
+  // todo(Léo): Why send unread counts computed before marking stuff as read?
```
Comment (leogrim) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationDeliveryCommander.scala:392]: This doesn't make sense to me, but then I don't think that method is actually in use.
@ryanpbrewster you may even be able to get rid of it as you kill the `NewMessage` stuff

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/MessagingCommander.scala [hunk 2]
```diff
@@ -63,34 +59,12 @@ class MessagingCommander @Inject() (
     messagingAnalytics: MessagingAnalytics,
     shoebox: ShoeboxServiceClient,
     airbrake: AirbrakeNotifier,
-    notificationRepo: NotificationRepo,
     basicMessageCommander: MessageFetchingCommander,
-    notificationCommander: NotificationCommander,
```
Comment (leogrim) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/MessagingCommander.scala:68]: Managed to remove the dependency on these guys. Hopefully we can remove `NotificationCommander`'s dependencies on the messaging repos.

File: kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationMessagingCommander.scala [hunk 3]
```diff
@@ -77,8 +77,8 @@ class NotificationMessagingCommander @Inject() (
   def sendUnreadNotificationsWith(notif: Notification, recipient: Recipient): Unit = {
     recipient match {
       case UserRecipient(user, _) =>
-        val unreadMessages = messagingCommander.getUnreadUnmutedThreadCount(user, Some(true))
-        val unreadNotifications = messagingCommander.getUnreadUnmutedThreadCount(user, Some(false))
+        val unreadMessages = messagingCommander.getUnreadUnmutedThreadCount(user)
+        val unreadNotifications = notificationCommander.getUnreadNotificationsCount(Recipient(user))
         webSocketRouter.sendToUser(user, Json.arr("unread_notifications_count", unreadMessages + unreadNotifications, unreadMessages, unreadNotifications))
         val pushNotif = MessageThreadPushNotification(ExternalId[MessageThread](notif.externalId.id), unreadMessages + unreadNotifications, None, None)
         pushNotifier.notifyUser(user, pushNotif, false)
```
Comment (leogrim) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/commanders/NotificationMessagingCommander.scala:84]: I have a feeling this method will go away as well, and we'll be able to kill `NotificationMessagingCommander` altogether.

**Comment by ryanpbrewster** [comment]
I'm going to try merging my PR with yours and see what happens.

**Pull request closed without merging**